src/components/PriceBar/index.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+
+import {
+  Container,
+  WrapperOuter,
+  TextWrapper,
+  Text,
+  PriceWrapper,
+  ColorText,
+  BoldText,
+  LightText,
+  ContainerFooter,
+  Wrapper,
+  RulerText,
+} from "./styles";
+
+import { HiOutlineZoomIn, HiOutlineZoomOut, HiOutlineInformationCircle } from "react-icons/hi";
+
+import { Switch } from "@chakra-ui/react";
+
+const PriceBar: React.FC = () => {
+  return (
+    <>
+      <Container>
+        {/* <OpenCloseButton /> */}
+        <WrapperOuter>
+          <TextWrapper>
+            <ColorText>#83DJ34</ColorText>
+            <Text>tile*2</Text>
+          </TextWrapper>
+          <TextWrapper>
+            <ColorText>#83DJ34</ColorText>
+            <Text>tile*2</Text>
+          </TextWrapper>
+          <TextWrapper>
+            <ColorText>#83DJ34</ColorText>
+            <Text>tile*2</Text>
+          </TextWrapper>
+          <TextWrapper>
+            <ColorText>#83DJ34</ColorText>
+            <Text>tile*2</Text>
+          </TextWrapper>
+          <TextWrapper>
+            <ColorText>#83DJ34</ColorText>
+            <Text>tile*2</Text>
+          </TextWrapper>
+          <TextWrapper>
+            <ColorText>#83DJ34</ColorText>
+            <Text>tile*2</Text>
+          </TextWrapper>
+        </WrapperOuter>
+
+        <PriceWrapper>
+          <BoldText>Estimated Budget: From $ 1647</BoldText>
+          <LightText>Please note: Final total price may very from estimated budget.</LightText>
+        </PriceWrapper>
+      </Container>
+
+      <ContainerFooter>
+        <Wrapper>
+          <HiOutlineZoomOut />
+          <HiOutlineZoomIn />
+        </Wrapper>
+
+        <Wrapper>
+          <Wrapper>
+            <RulerText>Ruler On</RulerText>
```

Comment:
This "Ruler On" text should be a label for the switch component so that clicking this text also changes the state of the switch. Please have a check on Chakra document switch section on how to wrap it with FormControl.
![image](https://user-images.githubusercontent.com/12379895/168466257-7487276b-32df-4bd0-9532-5a36ab47f97b.png)